### PR TITLE
Use quic by default

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR fixes https://github.com/consensus-shipyard/lotus/issues/171

### CODE REVIEW

**Review by adlrocha — APPROVED** [review]
LGTM. If you have tested the 4-node-net scripts and it works. Before you merge, we should also update the IPC scripts accordingly: https://github.com/consensus-shipyard/lotus/tree/spacenet/scripts/ipc.

This is what is used from the agent to deploy the infrastructure. Thanks :pray:

**Comment by dnkolegov** [comment]
I could not find where you configure tcp/udp libp2p address in IPC scripts. Could you point me?

**Comment by adlrocha** [comment]
We should grep here `udp` instead of `tcp`, that should be enough: https://github.com/consensus-shipyard/lotus/blob/f081f219b9205a1ce200b0866f250048bcaec7e4/scripts/ipc/run-subnet-docker.sh#L30 let me know if you need any follow-up.

File: scripts/ipc/src/root-single-validator.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ sleep 5
 eudico wallet import --as-default --format=json-lotus  /scripts/ipc/src/wallet.key
 echo "[*] Initializing validator config and adding validator"
 eudico mir validator config init
-validator_addr=`eudico mir validator config validator-addr | grep -vE '(/ip6/)' | grep -E '/ip4/127.0.0.1/tcp/1347'`
+validator_addr=`eudico mir validator config validator-addr | grep -vE '(/ip6/)' | grep -E '/ip4/127.0.0.1/udp/1347/quic'`
```
Comment (adlrocha) [inline scripts/ipc/src/root-single-validator.sh:14]: The default one is 1348: https://github.com/consensus-shipyard/lotus/blob/f081f219b9205a1ce200b0866f250048bcaec7e4/cmd/eudico/mirvalidator/cfg.go#L25 
```suggestion
validator_addr=`eudico mir validator config validator-addr | grep -vE '(/ip6/)' | grep -E '/ip4/127.0.0.1/udp/1348/quic'`
```

**Review by adlrocha — APPROVED** [review]
A slight change before merging. Out of a curiosity, have you tested it to see that it works?

**Comment by dnkolegov** [comment]
> A slight change before merging. Out of a curiosity, have you tested it to see that it works?

I ran the 4-node script, but not the ipc script.

**Comment by dnkolegov** [comment]
We expose these ports in the docker file
```
EXPOSE 1234
EXPOSE 1347
EXPOSE 1348
EXPOSE 2345
EXPOSE 3456
EXPOSE 1777
```

What ports are UDP? I believe at least 1347.

**Comment by adlrocha** [comment]
> What ports are UDP? I believe at least 1347.

If I am not mistaken, if you don't specify the protocol it is exposed for both UDP and TCP

**Comment by adlrocha** [comment]
> I ran the 4-node script, but not the ipc script.

I did the latter, so feel free to merge :)

**Comment by dnkolegov** [comment]
> > What ports are UDP? I believe at least 1347.
> 
> If I am not mistaken, if you don't specify the protocol it is exposed for both UDP and TCP

Only TCP. I send a commit, please review

**Review by adlrocha — APPROVED** [review]
LGTM

**Pull request merged**